extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelExtractor.java

```diff
@@ -191,12 +190,7 @@ public long getSubscriberCount() throws ParsingException {
                 throw new ParsingException("Could not get subscriber count", e);
             }
         } else {
-            // If there's no subscribe button, the channel has the subscriber count disabled
-            if (c4TabbedHeaderRenderer.has("subscribeButton")) {
-                return 0;
-            } else {
-                return -1;
-            }
+            return -1;
```

Comment:
why `-1` and not `ITEM_COUNT_UNKNOWN`?

Reply:
That constant comes from `ListExtractor`, so it is not related to this, I was wrong when talking about it above

Reply:
so i should remove it?